flow-typed/types/ui-component.js

```diff
@@ -1,11 +1,5 @@
 // @flow
-declare type KPUIComponent = {
-  label: string,
-  presets: Array<string>,
-  area: string,
+declare type KPUIComponent = KPPluginUIComponent & {
```

Comment:
all components are plugin component? no, so seems the type is incorrect. KPUIComponentOptions?